### PR TITLE
Added workflows and deployment

### PR DESCRIPTION
**Pull request by hussnain612**

### CODE REVIEW

File: .github/workflows/pull_request.yaml [hunk 1]
```diff
@@ -0,0 +1,91 @@
+name: Pull Request
+
+on:
+  pull_request:
+    branches:
+      - 'master'
```
Comment (abdulhaseeb2) [inline .github/workflows/pull_request.yaml]: * main

File: .github/workflows/push.yaml [hunk 2]
```diff
@@ -0,0 +1,100 @@
+name: Push
+
+on:
+  push:
+    branches:
+      - 'master'
```
Comment (abdulhaseeb2) [inline .github/workflows/push.yaml]: * main

**Review by abdulhaseeb2 — CHANGES_REQUESTED** [review]

File: deployments/kubernetes/templates/chart/values.yaml.tmpl [hunk 3]
```diff
@@ -0,0 +1,41 @@
+kubernetes:
+  host: https://kubernetes.default
+
+stakaterclouddocs-2.0:
+  labels:
+    group: com.stakater.platform
+    provider: stakater
+    version: {{ getenv "VERSION" }}
+  service:
+    name: "stakaterclouddocs-2.0"
```
Comment (MuneebAijaz) [inline deployments/kubernetes/templates/chart/values.yaml.tmpl]: do k8s object support this dot format? also you can mention it as stakakterclouddocs2 or stakaterclouddocs-v2.

File: deployments/kubernetes/templates/chart/Chart.yaml.tmpl [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+version: {{ getenv "VERSION" }}
+keywords:
+  - stakater
+  - cloud
+  - openshift
+home: https://github.com/stakater/stakater-cloud-docs-2.0
+maintainers:
+- name: Stakater
+  email: hello@stakater.com
+- name: Waleed Malik
```
Comment (MuneebAijaz) [inline deployments/kubernetes/templates/chart/Chart.yaml.tmpl]: Update name here as well

File: deployments/kubernetes/chart/stakaterclouddocs/values.yaml [hunk 5]
```diff
@@ -0,0 +1,37 @@
+stakaterclouddocs-2.0:
```
Comment (MuneebAijaz) [inline deployments/kubernetes/chart/stakaterclouddocs/values.yaml]: pls verify if chart names support dots format

**Comment by stakater-user** [comment]
@hussnain612 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/stakater-cloud-docs-2.0/commit/519e18da42bffed3326de89cb597d5561d5b2712/checks) has Failed!

File: .github/workflows/pull_request.yaml (hunk 1 above)
Comment (hussnain612) [reply .github/workflows/pull_request.yaml]: Updated

File: .github/workflows/push.yaml (hunk 2 above)
Comment (hussnain612) [reply .github/workflows/push.yaml]: Updated

File: deployments/kubernetes/templates/chart/values.yaml.tmpl (hunk 3 above)
Comment (hussnain612) [reply deployments/kubernetes/templates/chart/values.yaml.tmpl]: Updated with v2

File: deployments/kubernetes/templates/chart/Chart.yaml.tmpl (hunk 4 above)
Comment (hussnain612) [reply deployments/kubernetes/templates/chart/Chart.yaml.tmpl]: Updated

File: deployments/kubernetes/chart/stakaterclouddocs/values.yaml (hunk 5 above)
Comment (hussnain612) [reply deployments/kubernetes/chart/stakaterclouddocs/values.yaml]: Updated

**Comment by stakater-user** [comment]
@hussnain612 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/stakater-cloud-docs-2.0/commit/0ef1f9a69e854065cf35929294191c3e792a8c43/checks) has Failed!

**Comment by stakater-user** [comment]
@hussnain612 Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakater-cloud-docs-2.0:SNAPSHOT-PR-9-1c2b48e9`

**Review by MuneebAijaz — APPROVED** [review]

**Comment by stakater-user** [comment]
@hussnain612 Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakater-cloud-docs-2.0:v0.0.1`

**Pull request merged**